### ISSUE TITLE
Harvest Detail View failing to load because of a template loading error

### ISSUE THREAD

**Issue by mina-andajos** [LesFruitsDefendus/saskatoon-ng#252]
![image](https://user-images.githubusercontent.com/52796958/163462922-7f3182cd-7d26-4ce5-b536-89f772548a6f.png)
the templates aren't using `{% load i18n %}` which is causing an issue. I'm working on this issue right now.

**Comment by mina-andajos** [comment]
Traceback:
```sh
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/harvest/1/

Django Version: 3.2.12
Python Version: 3.9.11
Installed Applications:
['dal',
 'dal_select2',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'ckeditor',
 'leaflet',
 'sitebase',
 'member',
 'harvest',
 'rest_framework',
 'django_filters',
 'crispy_forms',
 'debug_toolbar',
 'django_extensions',
 'rosetta',
 'phone_field']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware']


Template error:
In template /mnt/d/niabc/Programming/Outreachy/Contributions/saskatoon-ng/saskatoon/sitebase/templates/app/detail_views/harvest/distribution/recipient_select.html, error at line 3
   Invalid block tag on line 3: 'translate'. Did you forget to register or load this tag?
   1 : {% load static %}
   2 : <select name="actor" id="recipient-select" class="form-control required">
   3 :     <option selected disabled> {% translate 'Owner' %} </option>
   4 :     <option value={{ property.owner.pk }}>{{ property.owner }}</option>
   5 :     <option selected disabled>{% translate 'Pickers' %}</option>
   6 :     {% for p in pickers %}
   7 :         <option value={{ p.pk }}>{{ p }}</option>
   8 :     {% endfor %}
   9 :     <option selected disabled>{% translate 'Organizations' %}</option>
   10 :     {% for o in organizations %}
   11 :     <option value={{ o.pk }}>{{ o }}</option>
   12 :     {% endfor %}
   13 :     <option selected disabled hidden>{% translate 'Select a Recipient' %}</option>


Traceback (most recent call last):
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]

During handling of the above exception ('translate'), another exception occurred:
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/core/handlers/base.py", line 204, in _get_response
    response = response.render()
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/rest_framework/renderers.py", line 167, in render
    return template.render(context, request=request)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 170, in render
    return self._render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loader_tags.py", line 195, in render
    return template.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 172, in render
    return self._render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loader_tags.py", line 195, in render
    return template.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 172, in render
    return self._render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/test/utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loader_tags.py", line 183, in render
    template = context.template.engine.select_template(template_name)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/engine.py", line 174, in select_template
    return self.get_template(template_name)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/loaders/base.py", line 29, in get_template
    return Template(
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/pure-cheekbones/anaconda3/envs/saskatoon-ng/lib/python3.9/site-packages/django/template/base.py", line 531, in invalid_block_tag
    raise self.error(

Exception Type: TemplateSyntaxError at /harvest/1/
Exception Value: Invalid block tag on line 3: 'translate'. Did you forget to register or load this tag?

```

**Comment by mina-andajos** [comment]
Looks like the problem is caused by afc293568eda73f238d022c3818498dcdfd3496c that was fixing translations.

**Comment by tristanlatr** [comment]
Fixed in #246, please go ahead and approve my PR, I'll be able to merge it after :)

**Comment by mina-andajos** [comment]
@tristanlatr sorry didn't notice you've opened a PR already
